utils/include/PlatformUtils.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace PlatformUtils
+{
+	bool GetListOfProcessIds(std::list<unsigned>& list_of_PIDs);
```

Comment:
why did you use list here? Why not vector for example?

Reply:
Sorry, forgot to change it when rewriting this method. Changed to vector<unsigned>. Also, container field in CContainerOfProcesses changed to list<CProcess>, 'cause we need to make a lot of erases in random places of container.